src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1934,7 +1938,18 @@ internal void WriteOutput(string outputString, bool includeNewLine)
         private void WriteExceptionToHost(RuntimeException e)
         {
             var psObject = PSObject.AsPSObject(e.ErrorRecord);
-            s_writeStreamProperty.SetValue(psObject, s_errorStreamValue);
+
+            // Used to write ErrorRecords to the Error stream so they are rendered in the console correctly.
+            if (VersionUtils.IsPS7OrGreater)
+            {
+                s_writeStreamProperty.SetValue(psObject, s_errorStreamValue);
+            }
+            else
+            {
+                var note = new PSNoteProperty("writeErrorStream", true);
```

Comment:
```suggestion
                var note = new PSNoteProperty("WriteErrorStream", true);
```

Reply:
I got that from [PowerShell](https://github.com/PowerShell/PowerShell/pull/8785/files#diff-670ce327a8d3fb9db99715904ac785ecL1979) but since it's case insensitive I can make this change.

Reply:
May as well use what PowerShell uses. Strange they did it that way